src/main/asciidoc/_chapters/ops_mgt.adoc

```diff
@@ -2521,93 +2517,38 @@ NOTE: WALs are saved when replication is enabled or disabled as long as peers ex
 [[rs.failover.details]]
 ==== Region Server Failover
 
-When no region servers are failing, keeping track of the logs in ZooKeeper adds no value.
-Unfortunately, region servers do fail, and since ZooKeeper is highly available, it is useful for managing the transfer of the queues in the event of a failure.
+When no region servers are failing, keeping track of the logs in hbase:replication table adds no value.
+However, in case of region server failure, we will manage the transfer of the queues based on hbase:replication.
 
-Each of the master cluster region servers keeps a watcher on every other region server, in order to be notified when one dies (just as the master does). When a failure happens, they all race to create a znode called `lock` inside the dead region server's znode that contains its queues.
-The region server that creates it successfully then transfers all the queues to its own znode, one at a time since ZooKeeper does not support renaming queues.
-After queues are all transferred, they are deleted from the old location.
-The znodes that were recovered are renamed with the ID of the slave cluster appended with the name of the dead server.
+When a region server fails, the HMaster of master cluster will trigger the SCP, and all replication queues on the failed region server will be claimed in the SCP.
```

Comment:
OK, I think the current ref guide also need to be updated...

I'm not sure which is the accruate version, but starting from a minor release for 2.x, we do not watch on the region server's znode any more, we use SCP to schedule claim queue operations...